odoo/addons/base/res/res_config.py

```diff
@@ -437,14 +441,15 @@ def _get_classified_fields(self):
                 {   'default': [('default_foo', 'model', 'foo'), ...],
                     'group':   [('group_bar', [browse_group], browse_implied_group), ...],
                     'module':  [('module_baz', browse_module), ...],
+                    'config':  [('config_plop', config_parameter), ...],
```

Comment:
Documentation is inconsistent with the above example.

Reply:
fixed